motoman_gp110_support/config/opw_parameters_gp110.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# Parameters for use with IK solvers which support OPW (Ortho-Parallel Wrist)
+# kinematic configurations, as described in the paper "An Analytical Solution
+# of the Inverse Kinematics Problem of Industrial Serial Manipulators with an
+# Ortho-parallel Basis and a Spherical Wrist" by Mathias Brandstötter, Arthur
+# Angerer, and Michael Hofbaur (Proceedings of the Austrian Robotics Workshop
+# 2014, 22-23 May, 2014, Linz, Austria).
+#
+# The moveit_opw_kinematics_plugin package provides such a solver.
+#
+opw_kinematics_geometric_parameters:
+    a1:  0.320
+    a2:  -0.235
+    b:   0.000
+    c1:  0.540
+    c2:  0.870
+    c3:  1.020
+    c4:  0.200
+opw_kinematics_joint_offsets: [0.0, 0.0, deg(-90.0), 0.0, 0.0, deg(180.0)]
```

Comment:
You've probably checked this, but that `deg(180.0)` at the end is unusual.

Or at least: it's the first time I see it in an OPW param file.